cypress/integration/profile/update.spec.js

```diff
@@ -65,8 +65,8 @@ describe(`profile/update (from login)`, () => {
     goToNextStep('Military Details');
 
     cy.get('input[name="branchOfService"]').should('have.value', 'army');
-    cy.get('input[name="yearsOfService"]').should('have.value', '3');
-    cy.get('input[name="payGrade"]').should('have.value', 'E20');
+    cy.findByLabelText('Years Of Service*').should('have.value', '3');
```

Comment:
looks like you missed a few `cy.get()` in here.

Reply:
Hi @kylemh, id is missing for several input fields. That is why I have retained few of those get() as testing-library findBy* is not working. Should we fix this in source code?

Reply:
Feel free to make any changes to data- attributes in the source code; however, I'm curious why `findByLabelText` wouldn't work?

Reply:
label for attribute will refer id attribute in input field. The id attribute is missing in several fields especially select type fields. That is why it is not working here.

Reply:
Ah, yes the `react-select` inputs have wild HTML output. It'd be very cool if we had a custom `cy` command for typing into a select/multiselect based on the label.

I can open a new issue for that, or you can resolve it in a separate PR if you're interested?!

Reply:
@kylemh I have made these changes. Will you be opening a new issue for this or shall I trigger the PR?

Reply:
Feel free to resolve the problem in this PR @logesr - I can count this is 2 separate issues resolved 🙏